tests/models/state/BaseStateProfile-test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * BaseStateProfile-test.js
+ * Created by Lizzie Salita 5/7/18
+ */
+
+import BaseStateProfile from "../../../src/js/models/v2/state/BaseStateProfile";
```

Comment:
Can you clean up this `import`? You should be able to use a relative path from `/src/js`